src/main/drivers/pwm_output.c

```diff
@@ -135,7 +136,9 @@ static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 
 void pwmWriteMotor(uint8_t index, uint16_t value)
 {
-    pwmWritePtr(index, value);
+    if (motors[index].enabled) {
+        pwmWritePtr(index, value);
+    }
```

Comment:
Either of those would work. I'd prefer preventing pmwMotors being enabled - that makes the code more transparent and might avoid a really hard to track down but in future.